docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -0,0 +1,91 @@
+project as your startup project to ensure the right things are built and
+deployed.
+
+- **VisualStudioSetup**: this project builds Roslyn.VisualStudio.Setup.vsix. It
+  contains the core language services that provide C# and VB editing. It also
+  contains the copy of the compiler that is used to drive IntelliSense and
+  semantic analysis in Visual Studio. Although this is the copy of the compiler
+  that's used to generate squiggles and other information, it's not the
+  compiler used to actually produce your final .exe or .dll when you do a
+  build. If you're working on fixing an IDE bug, this is the project you want
+  to use.
+- **CompilerExtension**: this project builds Roslyn.Compilers.Extension.vsix.
+  This deploys a copy of the command line compilers that are used to do actual
+  builds in the IDE. It only affects builds triggered from the Visual Studio
+  experimental instance it's installed into, so it won't affect your regular
+  builds. Note that if you install just this, the IDE won't know about any
+  language features included in your build. If you're regularly working on new
+  language features, you may wish to consider building both the
+  CompilerExtension and VisualStudioSetup projects to ensure the real build and
+  live analysis are synchronized.
+- **ExpressionEvaluatorPackage**: this deploys the expression evaluator and
+  result providers, the components that are used by the debugger to parse and
+  evaluate C# and VB expressions in the watch window, immediate window, and
+  more. These components are only used when debugging.
+- **VisualStudioInteractiveWindow**: this deploys the "base" interactive window
+  experience that is shared by Roslyn, Python, and other languages. This code
+  is core support only and doesn't include any language specific logic.
+- **VisualStudioInteractiveSetup**: this deploys the Roslyn (i.e. C# and VB)
+  specific parts of the interactive window. If you're working on the
+  interactive experience, this the project you want to use as your startup
+  project.
+
+The experimental instance used by Roslyn is an entirely separate instance of
+Visual Studio with it's own settings and installed extensions. It's also, by
+default, a separate instance than the standard "Experimental Instance" used by
+other Visual Studio SDK projects. If you're familiar with the idea of Visual
+Studio hives, we deploy into the RoslynDev root suffix.
+
+If you want to try your extension in your day-to-day use of Visual Studio, you
+can find the extensions you built in your Binaries folder with the .vsix extension.
+You can double-click the extension to install it into your
+main Visual Studio hive. This will replace the base installed version. You can
+uninstall your version and go back to the "real" version by going Tools >
+Extensions and Updates, finding your extension, and choosing Uninstall. Your
+extension should be marked with the "Experimental" flag.
```

Comment:
How is this?

> If you want to try your extension in your day-to-day use of Visual Studio, you
> can find the extensions you built in your Binaries folder with the .vsix
> extension. You can double-click the extension to install it into your main
> Visual Studio hive. This will replace the base installed version. Once it's
> installed, you'll see it marked as "Experimental" in Tools > Extensions and
> Updates to indicate you're running your experimental version. You can uninstall
> your version and go back to the "real" version choosing your version and
> clicking Uninstall.